lib/collection/benches/batch_search_bench.rs

```diff
@@ -36,7 +36,7 @@ fn create_rnd_batch() -> CollectionUpdateOperations {
         let vectors = only_default_vector(&vector);
         let point = PointStruct {
             id: (i as u64).into(),
-            vector: vectors.into(),
+            vector: Into::<VectorStruct>::into(vectors).into(),
```

Comment:
Can we replace all those `Into::<VectorStruct>::into()` with `VectorStruct::from()`?

It is both shorter and clickable.

Reply:
done